src/components/Block/Block.test.tsx

```diff
@@ -153,11 +146,27 @@ describe('<Block />', () => {
 
   describe('with negative push', () => {
     beforeEach(() => {
-      component = renderWithoutContext(<Block push={-2}>block content</Block>);
+      component = render(<Block push={-2}>block content</Block>);
     });
 
     it('matches its snapshot', () => {
       expect(component).toMatchSnapshot();
     });
   });
+
+  describe('within clickable', () => {
+    let rendered: Clickable;
+
+    beforeEach(() => {
+      rendered = renderIntoDocument(
+        <Clickable>
+          <Block>block content</Block>
+        </Clickable>,
+      ) as Clickable;
+    });
+
+    it('matches its snapshot', () => {
+      expect(findRenderedDOMComponentWithClass(rendered, 'y-block')).toMatchSnapshot();
+    });
+  });
```

Comment:
This is currently a bit of an integration test.

It might be better to mock the clickable context.

Reply:
if it's more work i'd just leave as-is